pyxlsb/workbook.py

```diff
@@ -53,7 +53,7 @@ def _parse(self):
         temp.seek(0, os.SEEK_SET)
       self.stringtable = StringTable(fp=temp)
     except KeyError:
-      pass
+      temp.close()
```

Comment:
That's a good catch. However, calling the `close` only for `KeyError` is too restrictive, many other things can go wrong here.

Maybe add a bare-except to close and then reraise the exception.

Reply:
done